demosplan/DemosPlanCoreBundle/Utilities/Pagination/DemosPlanArrayAdapter.php

```diff
@@ -37,7 +34,7 @@ public function getNbResults()
      *
      * @return array|Traversable
      */
-    public function getSlice($offset, $length)
+    public function getSlice($offset, $length): iterable
     {
```

Comment:
- what about adding int type on each parameter?
- we can remove the doc comment I feel

Reply:
[1ad309d](https://github.com/demos-europe/demosplan-core/pull/3130/commits/1ad309d0027bf94d4572813cf92b056934a13ce4)